src/components/settings/Settings.h

```diff
@@ -286,13 +321,22 @@ namespace Pinetime {
     private:
       Pinetime::Controllers::FS& fs;
 
-      static constexpr uint32_t settingsVersion = 0x0007;
+      static constexpr uint32_t settingsVersion = 0x0008;
+
+      // To enable disabling it during notification sleep, differentiate between
```

Comment:
this is true. `state` is weather or not the setting is enabled, and `enabled` is weather or not it's currently active. This is so that when the watch goes into notification sleep `enabled` is off so the screen turns off, but `state` is still set to true so that the screen stays on again after exiting the notification sleep.

 Any idea for either a better implementation or a better naming scheme for the variables are welcome

Reply:
Yeah, I think this is just the naming that is a bit confusing. Maybe we can try to find better names or at least clarify their meaning in comments?

Reply:
Resolved by simplifying the whole setup. There's no need to store the actual state of AOD in the settings file